dox/DoxyHome

```diff
@@ -788,8 +792,8 @@ RECURSIVE              = NO
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                = ./test_* \
-                         ~/.. ./.. .. ../.. ../../.. /home/ /home/j/ /home/j/dev
+EXCLUDE                = dox/books/reference_manual/main.dox \
```

Comment:
Consider dox/books/*/main.dox instead of naming them both.

Reply:
Neat trick!